opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -39,12 +39,15 @@
 //!
 //! [Resource]: crate::Resource
 
+#[allow(unreachable_pub)]
+#[allow(unused)]
 pub(crate) mod aggregation;
 pub mod data;
 mod error;
 pub mod exporter;
 pub(crate) mod instrument;
 pub(crate) mod internal;
+#[cfg(feature = "experimental_metrics_custom_reader")]
 pub(crate) mod manual_reader;
```

Comment:
We should probably just delete `manual_reader.rs` file instead of providing it under a feature. There doesn't seem to be any use case for it.